sympy/geometry/tests/test_plane.py

```diff
@@ -218,3 +218,13 @@ def test_dimension_normalization():
     assert p.perpendicular_plane(a, b) == Plane(Point3D(0, 0, 0), (1, 0, 0))
     assert Plane((1, 2, 1), (2, 1, 0), (3, 1, 2)
         ).intersection((2, 1)) == [Point(2, 1, 0)]
+
+def test_parameter_value():
+    p = Plane((0, 0, 0), (0, 0, 1), (0, 1, 0))
+
+    param = p.parameter_value((0,1,0))
+    # (0,cos(t),sin(t))
+    assert param == 0
```

Comment:
What does parameter = 0 mean for a plane, mathematically? A plane is described by two parameters. Also, I suggest harder tests than just 0s and 1s, with given point being also like that. For example, SymPy hangs if given
```
P = Plane(Point(1, 2, 3), Point(0, 0, 0), Point(4, 2, 1))
P.parameter_value((5, 4, 4))
```
Actually, this appears to be because `P.arbitrary_point()` returns some nonsense -- apparently, Plane.arbitrary_point() was already broken in 1.1.1 (and earlier).

Reply:
I understand that for a Plane, arbitrary_point is actually not arbitrary; it just traces a circle parameterized by t. Which implies that when t does not exist, it is incorrect to report that the point is not on the plane. 

It seems P.arbitrary_point is not yet ready to be built upon. Just look at what it returns for P in my example. At present, it seems prudent to not implement parameter_value for planes, and instead to review the situation with its arbitrary.point method.